lib/npm-installation-manager.ts

```diff
@@ -70,6 +70,14 @@ export class NpmInstallationManager implements INpmInstallationManager {
 
 	private installCore(packageName: string, pathToSave: string, version: string, dependencyType: string): IFuture<string> {
 		return (() => {
+			if(packageName.indexOf(".") === 0 || packageName.indexOf(".tgz") >= 0) {
```

Comment:
this will fail in case user passes path without `./` in front, for example in case I want to install local directory I can pass `./my-dir` or `my-dir`. Both will work